test/parallel/test-https-client-get-url.js

```diff
@@ -31,7 +29,12 @@ var server = https.createServer(options, function(req, res) {
 });
 
 server.listen(common.PORT, function() {
-  https.get('https://127.0.0.1:' + common.PORT + '/foo?bar');
+  https.get({
+    host: '127.0.0.1',
+    port: common.PORT,
+    path: '/foo?bar',
+    rejectUnauthorized: false
+  });
 });
```

Comment:
So it sounds like options are:
* Delete the test
* Keep the environment variable that disables certification verification
* Keep the test, but load a URL with a valid cert (for example, `https://letsencrypt.org/`)

Do the node tests run without internet access?

LMK what you prefer